.github/workflows/actions.yml

```diff
@@ -0,0 +1,38 @@
+name: deployment
+
+on:
+  push
+  # if u want to push only on master branch
+  # push:
+  #   branches:
+  #     - master
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest

```

Comment:
Linux should as default OS, isn’t it?

Reply:
Note from docs

![image](https://user-images.githubusercontent.com/64708421/165342912-9545ba1a-eec6-4469-b5e7-3805849293ae.png)

Reply:
yep, please test